File: 프로그래머스/0/181879. 길이에 따른 연산/길이에 따른 연산.java
class Solution {
    public int solution(int[] num_list) {
        int returnValue = 0;

        if (num_list.length > 10) {
            for (int val : num_list) {
                returnValue += val;
            }
        } else {
            returnValue = num_list[0];

            for (int i = 1; i < num_list.length; i++) {
                returnValue *= num_list[i];
            }
        }

        return returnValue;
    }
}

File: 프로그래머스/0/181879. 길이에 따른 연산/길이에 따른 연산.js
function solution(num_list) {
    var answer = 0;
    
    if (num_list.length >= 11) {
        for (const val of num_list) {
            answer += val
        }
    } else {
        answer = num_list[0]
        
        for (let i = 1; i < num_list.length; i++) {
            answer *= num_list[i]
        }
    }
    
    return answer;
}